rpc/src/server/router.rs

```diff
@@ -236,16 +246,6 @@ pub(crate) fn create_routes(tezedge_is_enabled: bool) -> PathTree<MethodHandler>
             "/chains/:chain_id/blocks/:block_id/context/constants",
             protocol_handler::context_constants,
         );
-        routes.handle(
```

Comment:
@adonagy @tizoc 
I see, we have constants for 009/010 finally, so I guess this 
```
routes.handle(
            hash_set![Method::GET],
            "/chains/:chain_id/blocks/:block_id/context/constants",
            protocol_handler::context_constants,
        );
```
can go outside of the if and can use rust implementation instead of protocol router

Reply:
Let's keep the constants routed for now. I was focusing mainly on the baking and endorsing rights. Those constants in tezos_messages are most likely incorrect. We could also serve the constatns RPC from the new storage, but only the constants used by the rights RPCs are properly tested, so I wouldn't risk it now. Will create a jira shortly.